goty-server/src/test/kotlin/com/aleinin/goty/game/IGDBGameSearchClientTest.kt

```diff
@@ -0,0 +1,67 @@
+package com.aleinin.goty.game
+
+import com.api.igdb.request.IGDBWrapper
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.kotlin.any
+
+@ExtendWith(MockitoExtension::class)
+internal class IGDBGameSearchClientTest {
+    @Mock
+    lateinit var igdbWrapper: IGDBWrapper
+
+    lateinit var client: IGDBGameSearchClient
+
+    @BeforeEach
+    fun setup() {
+        client = IGDBGameSearchClient(igdbWrapper, ObjectMapper())
+    }
+
+    @Test
+    fun `Should find main games by year`() {
```

Comment:
It's hard to tell for sure but I don't think these tests are adding much value. Regardless of what you pass in to `runTest` the assertion is the exact same.

Reply:
I agree. I refactored it. The pro is the tests are a bit more meaningful. The cons are its a bit more muddied with IGDB implementation details. With that said I think it is something worth testing and I can't think where else where this logic would be tested. I won't resolve this one yet because I'm curious what you think

Reply:
Yeah this looks much better. I think it's fine it is closely coupled with IGDB details because it's the IGDB client. Nice.